plugin.json

```diff
@@ -30,11 +30,20 @@
         },
         {
           "key": "EncryptionKey",
-          "display_name": "Encryption Key:",
+          "display_name": "At Rest Encryption Key:",
           "type": "generated",
-          "help_text": "The encryption key used to encrypt the authorization token.",
-          "regenerate_help_text": "Regenerates the encryption key for encrypting the authorization token. Regenerating the authorization invalidates your existing Confluence connection."
-      }
+          "help_text": "The encryption key used to encrypt stored API tokens.",
+          "placeholder": "",
+          "default": null,
+          "secret": true
+        },
+        {
+          "key": "AdminAPIToken",
+          "display_name": "Confluence Admin API Token",
+          "type": "text",
+          "help_text": "Set this [API token](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html) to get notified for confluence events when the user triggering the event is not connected to Confluence.\n**Note:** API token should be created using an admin Confluence account. Otherwise, the notification will not be delivered for the spaces/pages user does not have access.",
```

Comment:
Minor nit.  Can we change `Otherwise, the notification will not be delivered for the spaces/pages user does not have access.`

to 

`Otherwise, the notification will not be delivered for spaces/pages that the user does not have access to.`